spec.html

```diff
@@ -31976,13 +31976,13 @@ <h1>Pattern</h1>
           1. Return a new Abstract Closure with parameters (_str_, _index_) that captures _m_ and performs the following steps when called:
             1. Assert: Type(_str_) is String.
             1. Assert: ! IsNonNegativeInteger(_index_) is *true* and _index_ &le; the length of _str_.
-            1. If _Unicode_ is *true*, let _Input_ be a List consisting of the sequence of code points of ! StringToCodePoints(_str_). Otherwise, let _Input_ be a List consisting of the sequence of code units that are the elements of _str_. _Input_ will be used throughout the algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref>. Each element of _Input_ is considered to be a character.
+            1. If _Unicode_ is *true*, let _Input_ be ! StringToCodePoints(_str_). Otherwise, let _Input_ be a List containing the code units that are the elements of _str_. _Input_ will be used throughout the algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref>. Each element of _Input_ is considered to be a character.
             1. Let _InputLength_ be the number of characters contained in _Input_. This alias will be used throughout the algorithms in <emu-xref href="#sec-pattern-semantics"></emu-xref>.
             1. Let _listIndex_ be the index into _Input_ of the character that was obtained from element _index_ of _str_.
             1. Let _c_ be a new Continuation with parameters (_y_) that captures nothing and performs the following steps when called:
               1. Assert: _y_ is a State.
               1. Return _y_.
-            1. Let _cap_ be a List of _NcapturingParens_ *undefined* values, indexed 1 through _NcapturingParens_.
+            1. Let _cap_ be a List containing ~empty~ followed by _NcapturingParens_ *undefined* values.
```

Comment:
This is the only substantive change in this PR. Since it is unclear what "indexed 1 through *n*" means, I prepend an `~empty~` to put all the `*undefined*`s in the right place.

Reply:
cc @gibson042

Reply:
That's fair. How about we pull this change from both PRs and address it separately?

Reply:
Could you add comments at #2112 to explain what exactly you want pulled out? All of the changes in that PR relating to captures are just clarifications of the existing text (i.e., notes and reminders about 1-indexing). My preference is to leave those in even while anticipating a dedicated followup to pursue zero-indexing, because they improve the current state of affairs in which references to `_cap_[_k_]`/`_cap_[_n_]`/etc. are absurdly easy to misinterpret.